e2e-tests/setup.cfg

```diff
@@ -3,7 +3,9 @@ default_section = THIRDPARTY
 known_first_party = pages
 
 [tool:pytest]
-addopts = -n=auto --verbose -r=a --driver=Firefox
+addopts = -n=1 --verbose -r=a --driver=Firefox
+# we set -n=1 to help mitigate crashes running tests;
+# see https://bugzilla.mozilla.org/show_bug.cgi?id=1433503
```

Comment:
I think it would be better to just remove this option, so pytest-xdist is not used at all. If you need to have the option then I believe setting the value to zero is effectively the same as not having pytest-xdist in use.

Reply:
@davehunt That was my first thought, but I switched gears to make it more visually explicit/clear.  Happy to remove it and let Git blame/history elucidate.

Will wait for @willkg's take, and then fix accordingly.

Reply:
I'm +1 on simplifying things. We don't have many tests, so we should be fine with not parallelizing them. We can always re-add it later when there's a need.